.eslintrc.js

```diff
@@ -223,5 +224,18 @@ module.exports = {
 				],
 			},
 		],
+		// Disabled for now until we finish the migration
+		'wpcalypso/no-package-relative-imports': [
+			'off',
```

Comment:
Why not at least `warn`?

Reply:
I'd like to do more testing before enabling the rule in any form, to make sure people don't accidentally start adding `wp-calypso/` everywhere before we are 100% sure it works in all cases.